composer.json

```diff
@@ -15,13 +15,12 @@
     "laminas/laminas-cache-storage-implementation": "1.0"
   },
   "require-dev": {
-    "boesing/psalm-plugin-stringf": "^1.2",
```

Comment:
@boesing I dropped this, but I don't know if you are attached to it.

Please feel free to rollback this change, if it is an important component for you.

Reply:
Its not, if there are no psalm issues when removing it, I'm fine with that.
Did it report problems which were invalid?

Reply:
No, it was just not compatible with `vimeo/psalm:^5`, preventing the upgrade, for now.

Reply:
It definitely is 🤔 
Since may 2022 https://github.com/boesing/psalm-plugin-stringf/releases/tag/1.2.0

Reply:
Yep, shall revert and retry then